app/views/components/_markdown-editor.html.erb

```diff
@@ -1,17 +1,26 @@
+<%
+  textarea||= {}
```

Comment:
Missing spaces here (to be consistent with the Ruby style guide).

Reply:
Space added